content/en/docs/tasks/security/authorization/authz-jwt/index.md

```diff
@@ -0,0 +1,182 @@
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output, retry after a few seconds.
+Caching and propagation can cause a delay.
+{{< /warning >}}
+
+## Allow requests with valid JWT and list-typed claims
+
+1. The following command creates the `jwt-example` request authentication policy
+for the `httpbin` workload in the `foo` namespace. The policy allows the Envoy proxy in front of
+the `httpbin` workload to accept a JWT issued by `testing@secure.istio.io`.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "RequestAuthentication"
+    metadata:
+      name: "jwt-example"
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      jwtRules:
+      - issuer: "testing@secure.istio.io"
+        jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+    EOF
+    {{< /text >}}
+
+1. Verify that a request with an invalid JWT is denied.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -s -o /dev/null -H "Authorization: Bearer invalidToken" -w "%{http_code}\n"
+    401
+    {{< /text >}}
+
+1. Verify that a request without a JWT is allowed because there is no authorization policy.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+1. The following command creates the `require-jwt` authorization policy for the `httpbin` workload in the `foo` namespace.
+The policy requires all requests to the `httpbin` workload to have a valid JWT with
+`requestPrincipal` set to `testing@secure.istio.io/testing@secure.istio.io`.
+Istio constructs the `requestPrincipal` by combining the `iss` and `sub` of the JWT token
+with a `/` separator as shown.
+
+    {{< text syntax="bash" expandlinks="false" >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: require-jwt
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           requestPrincipals: ["testing@secure.istio.io/testing@secure.istio.io"]
+    EOF
+    {{< /text >}}
+
+1. Get the JWT that sets the `iss` and `sub` keys to the same value, `testing@secure.istio.io`.
+This causes Istio to generate the attribute `requestPrincipal` with the value,
+`testing@secure.istio.io/testing@secure.istio.io`.
+
+    {{< text syntax="bash" expandlinks="false" >}}
```

Comment:
use of `expandlinks="false"` is correct here.

Reply:
The website infrastructure does not handle the condition of a link to a file expansion while in the same block two consecutive `@` is used for non-linked purposes. In this case, you want that link expansion to occur. This would be an area for improvement, although it seems like such an edge case, as to not be worth the trouble.

Reply:
(no action needed).